internal/core/src/segcore/FieldIndexing.cpp

```diff
@@ -35,7 +35,7 @@ VectorFieldIndexing::VectorFieldIndexing(const FieldMeta& field_meta,
                                                field_index_meta,
                                                segcore_config,
                                                SegmentType::Growing)) {
-    index_ = std::make_unique<index::VectorMemIndex>(
+    index_ = std::make_unique<index::VectorMemIndex<float>>(
```

Comment:
what would happen if the vector field is binary vector?

Reply:
Nothing will happen, because T is not used currently. But it will soon be used to interface with knowhere's multi-type modules.